Controllers/controllers.js

```diff
@@ -0,0 +1,9 @@
+const { fetchCategories } = require("../Models/models.js")
+
+exports.returnCategories = (req, res) => {
+   fetchCategories().then((categories) => {
+       res.status(200).send(categories);
```

Comment:
just like the comment on the test, we should be structuring this as an object, with a key of categories as such
{ categories: [......]}